src/app/rpmostree-pkg-builtins.c

```diff
@@ -125,6 +156,7 @@ rpmostree_builtin_pkg_add (int            argc,
   glnx_unref_object RPMOSTreeSysroot *sysroot_proxy = NULL;
   g_autoptr(GPtrArray) packages_to_add =
     g_ptr_array_new_with_free_func (g_free);
+  g_autoptr(GPtrArray) local_packages_to_add = g_ptr_array_new ();
```

Comment:
The fds are closed in `pkg_change()`, but yeah they wouldn't be if an error occurs before that.